dlk/python/dlk/templates/src/func/impl/generic/quantized_conv2d_kn2row.cpp

```diff
@@ -50,24 +52,29 @@ void QuantizedConv2DKn2Row(const kn2row_input_t& input,
 
   Measurement::Start("quantized-kn2row");
 
-  auto kernel_ = MatrixView<QUANTIZED_PACKED_KERNEL, MatrixOrder::RowMajor>(
-      kernel.data(), oc * kh * kw, ic / 32);
-  auto input_ = MatrixView<QUANTIZED_PACKED, MatrixOrder::ColMajor>(
-      input.data(), ic / 16, ih * iw);
   auto output_ = MatrixView<BIN_CONV_OUTPUT, MatrixOrder::ColMajor>(
       p.device_output_buf, oc, ih * iw);
-
   if (kh == kw && kw == 3) {
-    unsigned bufsize = oc * kh * kw * ih * iw;
-    BIN_CONV_OUTPUT *kn2row_buf = new BIN_CONV_OUTPUT[bufsize]();
-    auto buf_ = MatrixView<BIN_CONV_OUTPUT, MatrixOrder::ColMajor>(
-        kn2row_buf, oc * kh * kw, ih * iw);
-
-    quantized_matrix_multiplication(kernel_, input_, buf_);
+    auto kernel_ = MatrixView<QUANTIZED_PACKED_KERNEL, MatrixOrder::RowMajor>(
```

Comment:
This definiton of `kernel_` and the line 71 looks duplicated. How about define this variable outside of if statement?